src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/AppViewModel.kt

```diff
@@ -21,8 +24,8 @@ data class DefaultViewModel(override val loggedIn: Boolean,
     override val appUrl = AppConfig()["app.url"]
     override val appEmail = AppConfig()["app.email"]
```

Comment:
Sorry to be pedantic but shall we then use the `appConfig` var instead of `AppConfig()` for these other props too for consistency?